src/events/AnalyticsEvents.ts

```diff
@@ -21,6 +21,11 @@ export interface IAnalyticsCustomEventArgs {
   customEvent: IAPICustomEvent;
 }
 
+export interface ICoveoUAEventArgs {
+  event: string;
```

Comment:
The `event` could be typed to `'CoveoCustomEvent' | 'CoveoClickEvent' | 'CoveoSearchEvent'` instead of generic string

Reply:
👍